src/main/java/phrasecount/Application.java

```diff
@@ -59,9 +57,13 @@ public static void configure(FluoConfiguration fluoConfig, Options opts) {
             PhraseMap.PcmUpdateObserver.class, String.class, Counts.class,
             opts.phraseCountMapBuckets));
 
-    // setup an export queue to to send phrase count updates to an Accumulo table
-    ExportQueue.configure(fluoConfig, new ExportQueue.Options(EXPORT_QUEUE_ID, AccumuloExporter.class.getName(), String.class.getName(), AccumuloExport.class.getName(), opts.exportQueueBuckets));
-    AccumuloExporter.setExportTableInfo(fluoConfig, EXPORT_QUEUE_ID,
-        new TableInfo(opts.instance, opts.zookeepers, opts.user, opts.password, opts.exportTable));
+    // setup an Accumulo export queue to to send phrase count updates to an Accumulo table
+    ExportQueue.configure(fluoConfig,
+                          new ExportQueue.Options(EXPORT_QUEUE_ID, PhraseExporter.class.getName(),
+                                                  String.class.getName(), Counts.class.getName(),
+                                                  opts.exportQueueBuckets).setExporterConfiguration(
+                              new AccumuloExporter.Configuration(opts.instance, opts.zookeepers,
```

Comment:
I think it may make this a little easier to read if the AccumuloExporter config were declared on a line before configuring the export queue.